pkg/controller/volume/persistentvolume/scheduler_binder_test.go

```diff
@@ -328,23 +328,19 @@ func (env *testEnv) assumeVolumes(t *testing.T, name, node string, pod *v1.Pod,
 		}
 	}
 
-	env.internalBinder.podBindingCache.UpdateBindings(pod, node, bindings)
-
 	pvcCache := env.internalBinder.pvcCache
 	for _, pvc := range provisionings {
 		if err := pvcCache.Assume(pvc); err != nil {
 			t.Fatalf("Failed to setup test %q: error: %v", name, err)
 		}
 	}
 
-	env.internalBinder.podBindingCache.UpdateProvisionedPVCs(pod, node, provisionings)
+	env.internalBinder.podBindingCache.UpdateBindings(pod, node, bindings, provisionings)
 }
 
 func (env *testEnv) initPodCache(pod *v1.Pod, node string, bindings []*bindingInfo, provisionings []*v1.PersistentVolumeClaim) {
 	cache := env.internalBinder.podBindingCache
-	cache.UpdateBindings(pod, node, bindings)
-
-	cache.UpdateProvisionedPVCs(pod, node, provisionings)
+	cache.UpdateBindings(pod, node, bindings, provisionings)
```

Comment:
do we need to add UT for UpdateBindings with provisionings param nil or few different values of it ?

Reply:
Sure, UpdateBindings/UpdateProvisionedPVCs has been tested in  [scheduler_binder_cache_test.go](https://github.com/kubernetes/kubernetes/blob/f4e4a69aefe53ca8cd4d3fc954beba7700c83d1a/pkg/controller/volume/persistentvolume/scheduler_binder_cache_test.go#L63), we can add more cases.